### DISCUSSION TITLE
How with Auth::loginUsingId in seeder to set spatie/permission access?

### DISCUSSION THREAD

**Discussion by PetroGromovo** [spatie/laravel-permission#2207] [Q&A]
On laravel 9 site I fill some init data in seeders
and I use line :

    Auth::loginUsingId($defaultAdminId);

and in model_has_permissions table I have a row referenced $defaultAdminId and Admin
permission in permissions table

But later checking is logged is admin

    if ( Auth::user()->can(ACCESS_ADMIN) ) {   // Is Admin
        ...

    
The checks above does not work . I suppose method Auth::loginUsingId( does not know anything about 
model_has_permissions table . If there is a way to set spatie/permission access after calling Auth::loginUsingId( ?
                         
    "laravel/framework": "^9.26.1",
    "spatie/laravel-permission": "^5.5.5"
    "laravel/passport": "^10.4.1",

    
Thanks in advance!

**Comment by erikn69** [comment]
Try `Auth::user()->load('permissions');`  after login
https://github.com/spatie/laravel-permission/blob/67e54e2c7941d7aaa93c207549cdb7c5a44c68f5/src/Traits/HasPermissions.php#L350
